secretsauce/apps/portal/models.py

```diff
@@ -29,86 +29,84 @@ def __str__(self):
         return self.name
 
 class ModelTag(models.Model):
-    name = models.CharField(max_length=200)
+    name = models.CharField(max_length=200, unique=True, primary_key=True)
     prediction_model = models.ManyToManyField(
         PredictionModel,
-        related_name="tags"
+        related_name='model_tags',
     )
```

Comment:
What is ModelTag for again?

Reply:
ModelTag is for example "Neural Networks" or "Decision Trees", its like a tag to sort different types of Models by.